documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -141,6 +141,7 @@ func parseFlags() *config {
 	)
 	flag.StringVar(&cfg.listenAddr, "web.listen-address", ":9201", "Address to listen on for web endpoints.")
 	flag.StringVar(&cfg.telemetryPath, "web.telemetry-path", "/metrics", "Address to listen on for web endpoints.")
+	flag.StringVar(&cfg.logLevel, "log.level", "debug", "The log level(error/warn/info/debug) to be set for promlog. ")
```

Comment:
Users won't know what "promlog" is - I'd use the same flag description as in Prometheus itself: https://sourcegraph.com/github.com/prometheus/prometheus@2bd510a63e48ac6bf4971d62199bdb1045c93f1a/-/blob/vendor/github.com/prometheus/common/promlog/flag/flag.go#L26

Reply:
(+ extra whitespace at end of flag description)